lib/graphql/metrics/instrumentation.rb

```diff
@@ -47,7 +47,9 @@ def after_query(query)
       private
 
       def query_present_and_valid?(query)
-        query.valid? && query.document.to_query_string.present?
+        # Check for selected_operation as well for graphql 1.9 compatibility
+        # which did not reject "empty" documents in its parser.
+        query.valid? && query.selected_operation
```

Comment:
Should we change to `!query.selected_operation.nil?` instead, so that we we always return a boolean?

Reply:
Oh yeah I guess 👍

Reply:
Also realized that ActiveSupport wasn't an actual dependency of this project so it was a good side-effect to remove `present?`